Source/Atomic/UI/UI.cpp

```diff
@@ -150,7 +150,7 @@ void UI::HandleExitRequested(StringHash eventType, VariantMap& eventData)
 
 void UI::Shutdown()
 {
-    SetInputDisabled(true);
+    //SetInputDisabled(true);
```

Comment:
Yes, if you think it is not necessary.  

If we're going to comment out code, but leave it in, we should make a note why.  

BTW, I don't have anyone reviewing my pull requests at the moment, so stuff like this may slip in :)